app/controllers/appropriate_bodies/process_batch/actions_controller.rb

```diff
@@ -21,7 +21,9 @@ def create
 
           redirect_to ab_batch_action_path(@pending_induction_submission_batch)
         else
-          @pending_induction_submission_batch = csv_data
+          csv_data.errors.each do |error|
```

Comment:
How about creating a service class with these actions so we avoid fat controllers?